ui/src/file_upload.tsx

```diff
@@ -103,13 +112,14 @@ const convertMegabytesToBytes = (bytes: F) => bytes * 1024 * 1024
 export const
   XFileUpload = ({ model }: { model: FileUpload }) => {
     const
-      { name, label, file_extensions, max_file_size, compact, height, max_size, multiple } = model,
+      { name, label, file_extensions, max_file_size, compact, height, max_size, multiple, required } = model,
       [isDragging, setIsDragging] = React.useState(false),
       [files, setFiles] = React.useState<File[]>([]),
       [fileNames, setFileNames] = React.useState<S>(''),
       [percentComplete, setPercentComplete] = React.useState(0.0),
       [error, setError] = React.useState(''),
       [successMsg, setSuccessMsg] = React.useState(''),
+      { semanticColors: { errorText: errorTextColor } } = Fluent.useTheme(),
```

Comment:
We could maybe make this a CSS var (`--errorText`) and set it on theme change. Would make also future code simpler.

Reply:
It is possible to achieve in two ways. Do you mean to add `semanticColors: {errorText: '#xxxxxx'}` into each theme in the `const themes: Dict<PredefinedTheme>` object in `theme.ts`? Or just get the `semanticColors.errorText` from theme after it loads/changes and then set it as css variable? 
Because if we go with second option I need to do it in `theme.ts` where theme is changed and in `index.ts` where theme is first loaded.

Reply:
>Or just get the semanticColors.errorText from theme after it loads/changes and then set it as css variable?

this one